lib/httparty/connection_adapter.rb

```diff
@@ -69,10 +70,11 @@ def initialize(uri, options = {})
 
     def connection
       host = clean_host(uri.host)
+      default_port = uri.scheme == 'https' ? 443 : 80
```

Comment:
The issue is that sometimes people run https on non-443 port. I've had people complain when I assume the port in the past.